### PR TITLE
Workaround the issue as long as BZ2228820 is open

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

**Review by ColeHiggins2 — APPROVED** [review]
Looks good to me

**Comment by ColeHiggins2** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_organization.py -k test_positive_create

File: tests/foreman/api/test_organization.py [hunk 1]
```diff
@@ -77,7 +78,10 @@ def test_positive_create(self):
             headers={'content-type': 'text/plain'},
```
Comment (Gauravtalreja1) [inline tests/foreman/api/test_organization.py:78]: For workaround here, wouldn't it better to change content-type instead when bz is open with `is_open` and not checking UNSUPPORTED_MEDIA_TYPE, wdyt?

Comment (lhellebr) [reply tests/foreman/api/test_organization.py:78]: As it is now, we're testing that UNSUPPORTED_MEDIA_TYPE is being returned, or at least ISE 500 if the BZ is open (as opposed to returning something even more wrong, e.g. 200 OK).

Comment (Gauravtalreja1) [reply tests/foreman/api/test_organization.py:78]: IMO using any other content-type's isn't wrong, and I feel its better option to test something than checking if we hit UNSUPPORTED_MEDIA_TYPE/ISE :)

Its upto you how you want to test it, so its non-blocking.

**Review by Gauravtalreja1 — APPROVED** [review]

**Comment by Gauravtalreja1** [comment]
Why you're looking for additional tier1 and tier2 reviews on this? :)

**Comment by adarshdubey-star** [comment]
Please add the description and we can merge this!

**Comment by lhellebr** [comment]
@Gauravtalreja1, because nobody merged and I shouldn't self-merge.

**Comment by lhellebr** [comment]
@adarshdubey-star there's both PR name and commit message, why do you think description is necessary? Is something unclear based on the commit message?

**Comment by adarshdubey-star** [comment]
> @adarshdubey-star there's both PR name and commit message, why do you think description is necessary? Is something unclear based on the commit message?

Nope just a good to have thing :)

**Pull request merged**